week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(sales) {
+  return sales * 0.2 + sales;
+ }
```

Comment:
you did fine here, you could have done this perfectly by storing the price multiplied by the tax in one variable then the sum in another variable, and return that variable